lib/tasks/webpacker/check_yarn.rake

```diff
@@ -1,15 +0,0 @@
-namespace :webpacker do
-  desc "Verifies if yarn is installed"
-  task :check_yarn do
-    required_yarn_version = "0.20.1"
```

Comment:
If slowness is the reason to remove it, do you think adding option to disable checking, will be better than removing it?